deploy/helm/clickhouse-operator/README.md

```diff
@@ -25,31 +25,31 @@ For upgrade please install CRDs separately:
+| additionalResources | list | `[]` | list of additional resources to create (processed via `tpl` function), useful for create ClickHouse clusters together with clickhouse-operator. check `kubectl explain chi` for details |
+| affinity | object | `{}` | affinity for scheduler pod assignment, check `kubectl explain pod.spec.affinity` for details |
+| configs | object | check the `values.yaml` file for the config content (auto-generated from latest operator release) | clickhouse operator configs |
```

Comment:
`helm/clickhouse-operator/README.md` fully regenerated via `helm-docs`

Are you sure it will work properly?